docs/user/disconnected-cluster.md

```diff
@@ -1,184 +1,91 @@
 # Installing a Flight Control in a Disconnected Cluster Environment
 
-> 📌 Please note, that version `0.4.0-rc3` are used for demonstration purposes only
+## Mirroring images
 
-> 📌 All `sha256` are valid for `0.4.0-rc3` version only
-
-## Log in
-
-### login to hub
+### Mirroring flightctl images
 
 ```shell
-ssh root@<hub-hostname>
+export FCTL_VERSION=<required_flightctl_version>
+export LOCAL_REGISTRY=<registry-hostname>:<registry-port>
 ```
 
-### all operations should be executed from `kni` user
+> 📌 Please note, that you may need to provide credentials for your local registry via `oc image mirror -a <creds_file>`.
 
 ```shell
-su - kni
+for component in api periodic ui worker; do oc image mirror quay.io/flightctl/flightctl-$component:${FCTL_VERSION} ${LOCAL_REGISTRY}/flightctl/flightctl-$component:${FCTL_VERSION}; done
 ```
 
-### login to cluster
+### Mirroring other images
 
 ```shell
-export KUBECONFIG=clusterconfigs/auth/kubeconfig
-```
+oc image mirror quay.io/keycloak/keycloak:25.0.1 ${LOCAL_REGISTRY}/keycloak/keycloak:25.0.1
 
-## Configure mirroring `icsp`
+oc image mirror quay.io/openshift/origin-cli:4.20.0 ${LOCAL_REGISTRY}/openshift/origin-cli:4.20.0
 
-### check whether `quay.io` mirror is configured in `icsp`
+oc image mirror quay.io/sclorg/postgresql-16-c9s:20250214 ${LOCAL_REGISTRY}/sclorg/postgresql-16-c9s:20250214
 
-```shell
-oc get imagecontentsourcepolicy image-policy -o yaml
+oc image mirror docker.io/redis:7.4.1 ${LOCAL_REGISTRY}/redis:7.4.1
 ```
 
-look for
-
-```yaml
-  - mirrors:
-    - <registry-hostname>:5000
-    source: quay.io
-```
+The mirroring instructions apply for partially disconnected environment, where from your management hub you can reach the internet.
+For fully disconnected environments, you will need to mirror images to a disk or USB stick on a connected machine and bring the mirrored content to the disconnected management hub. Then mirror the content to a local registry.
```

Comment:
Not sure why u call it 'partially disconnected env'.  The mirroring step is done on some machine, either by cloning from the internet or from usb-drive. But the focus is the cluster itself (it's nodes). i.e. external requests to internet are blocked, and only allowed to local registry ip/port.  and I can call it 'full disconnected env'.

Reply:
Ok, I rephrased the docs. Let me know if it is better now :)